authentication/certificate-types-descriptions.adoc

```diff
@@ -0,0 +1,47 @@
+[id="ocp-certificates"]
+= Certificate types and descriptions
+include::modules/common-attributes.adoc[]
+:context: ocp-certificates
+
+toc::[]
+
+== Certificate validation
+
+{product-title} monitors certificates for proper validity, for the cluster
+certificates it issues and manages. The {product-title} alerting framework has
+rules to help identify when a certificate issue is about to occur. These rules
+consist of the following checks:
+
+* API server client certificate expiration is less than five minutes.
+
+include::modules/user-provided-certificates-for-api-server.adoc[leveloffset=+1]
```

Comment:
@ahardin-rh maybe, based on @deads2k comments in our email thread we could add a category called `control plane certificates`.   That could state something like 

- managed by the system/rotated automatically
- located in {a, b, c} namespaces
- Note with known issue of expiration during certain corner cases with link to recovery of control plane cert doc for good measure

wdyt?

Reply:
@pweil- Sounds good to me!

Reply:
@pweil- The draft is updated. What specific namespaces should be listed? Thanks!

Reply:
Thanks Ashley.  Based on our discussion with @deads2k it sounds like these can be listed:

openshift-config-managed
openshift-kube-apiserver
openshift-kube-apiserver-operator
openshift-kube-controller-manager
openshift-kube-controller-manager-operator
openshift-kube-scheduler